frontend/Components/Login.js

```diff
@@ -0,0 +1,210 @@
+        </View>
+    );
+};
+
+const Styles = StyleSheet.create({
+    device: {
+        flexDirection: 'row',
+        backgroundColor: '#dffaef',
+        alignItems: 'flex-start'
+    },
+
+    container: {
+        marginHorizontal: '5%',
+        flex: 2,
+        justifyContent: 'flex-start',
+        backgroundColor: "#dffaef",
+        height: '100%'
+    },
+
+    logo: {
+        height: '15%',
+        width: '54%',
+        resizeMode: 'contain'
+    },
+
+    imgContainer: {
+        marginTop: "5%",
+        marginHorizontal: '5%',
+        flex: 4,
+        justifyContent: 'center',
+        backgroundColor: "#dffaef",
+        height: '100%'
+    },
+
+    themeImage: {
+        resizeMode: 'contain',
+        height: '100%',
+        borderRadius: 38,
+    },
+
+    title: {
+        fontSize: 36,
+        fontWeight: 'bold',
+    },
+
+    titleContainer: {
+        width: '100%',
+        height: '20%',
+        justifyContent: 'center',
+        paddingLeft: Platform.OS !== 'web' ? '10%' : '40%',
+        marginTop: '3%'
+    },
+
+    textBoxContainer: {
+        backgroundColor: 'white',
+        height: Platform.OS === 'web' ? '15%' : '8%',
+        marginVertical: '5%',
+        borderRadius: 10,
+        padding: '2%',
+    },
+
+    textBox: {
+        paddingHorizontal: '3%',
+        fontSize: 16,
+    },
+
+    button: {
+        marginVertical: '5%',
+        backgroundColor: '#0066F9',
+        borderRadius: 20,
+        alignItems: 'center',
+        justifyContent: 'center',
+        height: Platform.OS === 'web' ? '13%' : '8%'
+    },
+
+    buttonText: {
+        color: 'white',
+        fontSize: 20,
+        fontWeight: 'bold',
+    },
+
+    farLink: {
+        marginVertical: '4%',
+        marginLeft: '5%',
+        width: '50%'
+    },
+
+    lastLine: {
+        marginVertical: '4%',
+        justifyContent: 'center',
+        alignItems: 'center',
+    },
+
+    smallLink: {
+        fontWeight: 'bold',
+        textDecorationLine: 'underline',
+        fontSize: 14
+    },
+
+    inLineLink: {
+    },
+
+    mediumLink: {
+        fontWeight: 'bold',
+        textDecorationLine: 'underline',
+        fontSize: 16
+    }
+
+});
+
+const logoPath = require('../assets/images/LOGO.png'); 
+
+function MakeTextInput(props) {
+    const [value, onChangeText] = React.useState('');
+
+    return (
+        <View style={Styles.textBoxContainer}>
+            <Text style={{fontSize:16,margin:3}}>{props.type}</Text>
+            <TextInput
+                style={Styles.textBox}
+                placeholder={'Enter your ' + props.type}
+                placeholderTextColor={'gray'}
+                secureTextEntry={props.type==='Password'}
+                onChangeText={text => {
+                    onChangeText(text)
+                    console.log(props.type + ": " + text)
+                    }
+                }
+                value={value}
+            />
+        </View>     
+    );
+  }
```

Comment:
This is basically a component, so it should be a separate file especially that we reuse it again in signup form and maybe we use later somewhere else.

Although components are functions we shouldn't name them like so, because we use them in a declarative way like `<Input name={name}>` similar to how html tags works, so this should be named something along the lines of Input, Field, FormInput or something similar. Choose whatever name you find suitable but in a declarative way.

One other thing i would like to suggest is to avoid having specific behavior based on the props based